README.md

```diff
@@ -24,100 +24,56 @@ At [Viaduct](https://www.viaduct.ai/), we manage our Kubernetes resources via th
 
 
 ## Requirements
-- [Go](https://github.com/golang/go)
-- [kustomize](https://github.com/kubernetes-sigs/kustomize/) built with Go (See [details below](#kustomize-go-plugin-caveats))
+- [kustomize](https://github.com/kubernetes-sigs/kustomize/)
 - [SOPS](https://github.com/mozilla/sops)
 - gpg
 
 
 ## Installation Options
 
-### Go Plugin
-
-KSOPS was originally developed as a [kustomize Go plugin](https://kubernetes-sigs.github.io/kustomize/guides/plugins/#go-plugins). Up until *v2.2.0* this was the only installation option. To install, follow steps 0-3 of the [Getting Started section](#getting-started) and then run `make install`.
-
 ### Exec Plugin
 
-[kustomize exec plugins](https://kubernetes-sigs.github.io/kustomize/guides/plugins/#exec-plugins) offers a simpler installation and dependency management alternative to [kustomize Go plugin](https://kubernetes-sigs.github.io/kustomize/guides/plugins/#go-plugins). By popular demand, we now offer support for KSOPS as a kustomize exec plugin.
-
-Currently, the new KSOPS exec plugin is opt-in. It is installed as a new plugin, `ksops-exec`, when you run `make install`. To switch a manifest to use the new exec plugin, you can simply change the `kind` in the generator manifest.
-
-```yaml
-apiVersion: viaduct.ai/v1
-kind: ksops-exec
```

Comment:
Are we keeping compat for this old `kind`?

Reply:
@daurnimator My intention was to drop support for it, but understand that it will cause issues upgrading to this release. Would it be helpful if I keep the old `kind` in `v3.0.0` and drop it in a subsequent release for `v3.1.0`?

Reply:
> Would it be helpful if I keep the old `kind` in `v3.0.0` and drop it in a subsequent release for `v3.1.0`?

probably; though I think you could probably keep it until 4.0.0, it should be a low-cost bit of compatibility.